src/screens/ProfileScreen/elements/TextWithEmoji.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import {Text, StyleSheet} from 'react-native';
 import emojiRegex from 'emoji-regex';
 import {COLORS} from '../../../utils/theme';
+import { fonts } from '../../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `fonts` has been added. Please ensure that the `fonts` object is correctly exported from the `utils/fonts` module and contains the required font styles.

```diff
+ import { fonts } from '../../../utils/fonts';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->